src/interpreter/plugin/SimpleArithmertic.ts

```diff
@@ -0,0 +1,217 @@
+/**
+ * @license
+ * Copyright (c) 2020 Handsoncode. All rights reserved.
+ */
+
+import {InternalNoErrorCellValue, InternalScalarValue, SimpleCellAddress} from '../../Cell'
+import {ProcedureAst} from '../../parser'
+import {divide} from '../ArithmeticHelper'
+import {ArgumentTypes, FunctionPlugin} from './FunctionPlugin'
+
+export class SimpleArithmerticPlugin extends  FunctionPlugin {
+  public static implementedFunctions = {
+    'ADD': {
+      method: 'add',
+      parameters: [
+        { argumentType: ArgumentTypes.NUMBER },
+        { argumentType: ArgumentTypes.NUMBER },
+      ],
+    },
+    'CONCAT' : {
```

Comment:
We already have CONCATENATE in text functions. Maybe we can have this as an alias?

Reply:
CONCATENATE is to CONCAT as SUM is to ADD
(many args vs 2 args)

Reply:
Yup, this is not the Excel CONCAT and CONCATENATE

Reply:
Good point, sorry!